Celeste.Mod.mm/Patches/SaveData.cs

```diff
@@ -88,7 +88,8 @@ public class patch_SaveData : SaveData {
             get {
                 if (LevelSet == "Celeste")
                     return UnlockedAreas_Unsafe;
-                return LevelSetStats.AreaOffset + LevelSetStats.UnlockedAreas;
+                var stats = LevelSetStats;
```

Comment:
I think in Everest's code style we generally don't use `var` except for very long type names.